modal/_function_utils.py

```diff
@@ -53,6 +53,21 @@ def package_mount_condition(filename):
     return os.path.splitext(filename)[1] in [".py"]
 
 
+def entrypoint_only_package_mount_condition(entrypoint_file):
```

Comment:
If the `__init__.py` refers to other files/modules that we haven't mounted, does that cause problems?